packages/types/src/derived.ts

```diff
@@ -1,9 +1,26 @@
+/* eslint-disable jsdoc/require-jsdoc */
```

Comment:
I've added this because otherwise eslint requires me to add JSDocs for `state`, `actions` & `libraries` which in my opinion makes it actually harder to understand this type.